lib/kinematics/src/rotation3.cpp

```diff
@@ -45,4 +56,16 @@ Matrix3d Rotation3::GetRotationMatrix() const {
   return Rm;
 }
 
+Vector3d Rotation3::Rotate(const Vector3d &px) const {
+    Matrix3d Rm = this->GetRotationMatrix();
```

Comment:
When you start a new line, only use two spaces.
  Matrix3d Rm = this->GetRotationMatrix();

Use 4 spaces only when you have to split a long sentence into multiple lines.